your-basic-web-server.js

```diff
@@ -7,9 +7,15 @@
     1) directly require and create an instance of express in this file
      
      Use port 1337 for this server instance
+*/
+   const express = require('express')
+   const app = express()
 
+   app.listen(3002, function() {
```

Comment:
Looks like port 1337 was blocked for you. I will share a link on slack if you want to check if anything happens to be running on that port on your computer.